### PR TITLE
created basic component file setup and got headers basic setup done

### PR DESCRIPTION
**Pull request by Euniceem**

### CODE REVIEW

File: src/app/services/backend.service.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+
```
Comment (taesup) [inline src/app/services/backend.service.ts]: Remove empty line

File: src/shared/header/header.component.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
```
Comment (taesup) [inline src/shared/header/header.component.ts]: Remove empty line.

File: src/styles.scss [hunk 3]
```diff
@@ -1 +1,8 @@
 /* You can add global styles to this file, and also import other style files */
+html, body {
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  font-family: 'Playfair Display', serif;
+
```
Comment (taesup) [inline src/styles.scss]: Remove empty line.

File: src/app/pages/profile/profile.component.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.scss']
+})
+
```
Comment (taesup) [inline src/app/pages/profile/profile.component.ts]: Remove empty line.

File: src/app/pages/login/login.component.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: '../login/login.component.html',
+  styleUrls: ['../login/login.component.scss']
+})
+
```
Comment (taesup) [inline src/app/pages/login/login.component.ts]: Remove empty line

File: src/app/pages/home/home.component.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+
```
Comment (taesup) [inline src/app/pages/home/home.component.ts]: Remove empty line

File: src/app/pages/contact/contact.component.ts [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: '../contact/contact.component.html',
+  styleUrls: ['../contact/contact.component.scss']
+})
+
```
Comment (taesup) [inline src/app/pages/contact/contact.component.ts]: Remove empty line.

File: src/app/pages/add-contact/addContact.component.ts [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './addContact.component.html',
+  styleUrls: ['./addContact.component.scss']
+})
+
```
Comment (taesup) [inline src/app/pages/add-contact/addContact.component.ts]: Remove empty line.

**Review by taesup — CHANGES_REQUESTED** [review]

File: src/app/pages/contact/contact.component.ts [hunk 9]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: '../contact/contact.component.html',
```
Comment (taesup) [inline src/app/pages/contact/contact.component.ts]: These should just be `./contact.component.html` same for the scss file.

File: src/app/pages/login/login.component.ts [hunk 10]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: '../login/login.component.html',
```
Comment (taesup) [inline src/app/pages/login/login.component.ts]: no need to go up a directory here.

**Review by taesup — CHANGES_REQUESTED** [review]

**Review by melsoriano — APPROVED** [review]

**Pull request merged**